backend/src/main/java/org/eclipse/tractusx/puris/backend/erp_adapter/controller/ErpAdapterController.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2024 Volkswagen AG
+ * Copyright (c) 2024 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.eclipse.tractusx.puris.backend.erp_adapter.controller;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import io.swagger.v3.oas.annotations.Parameter;
+import lombok.extern.slf4j.Slf4j;
+
+import org.eclipse.tractusx.puris.backend.erp_adapter.logic.service.ItemStockErpAdapterService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Date;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("erp_adapter")
+@Slf4j
+public class ErpAdapterController {
+
+    @Autowired
+    private ItemStockErpAdapterService itemStockErpAdapterService;
+
+    @PutMapping
+    public ResponseEntity<?> putMethod(
+        @RequestParam("request_id") UUID requestId,
```

Comment:
Please add a short swagger doc. We did similar [here](https://github.com/eclipse-tractusx/puris/blob/main/backend/src/main/java/org/eclipse/tractusx/puris/backend/stock/controller/StockViewController.java#L531) but without params

Reply:
Done

Reply:
BTW: I also made a specific mention for a 409 response type in case we receive a response for a request-id that was answered to previously. 

PS: I also simplified things a bit more for testing. Now you don't need to copy the request-id from the logs of the customer-backend. Now the ErpAdapterRequestService will generally return you the mocked response created in the DataInjectionCommandLineRunner. That means, when testing you can basically use any syntactically correct UUID as a request-id parameter on that endpoint.

Reply:
Good idea!